src/pages/Inventory.page.js

```diff
@@ -1,15 +1,46 @@
 import { BaseSwagLabPage } from './BaseSwagLab.page';
 
 export class InventoryPage extends BaseSwagLabPage {
+
     url = '/inventory.html';
 
-    headerTitle = this.page.locator('.title');
+    get headerTitle() { return this.page.locator('.title')};
+
+    get inventoryItems() {return this.page.locator('.inventory_item')};
+
+    get inventoryItemName() { return this.page.locator('.inventory_item_name')};
 
-    inventoryItems = this.page.locator('.inventory_item');
+    get inventoryDescription() { return this.page.locator('.inventory_item_desc')};
 
-    addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
+    get inventoryPrice() {return this.page.locator('.inventory_item_price')}; 
 
-    async addItemToCartById(id) {
+    get addItemToCartButton() {return this.page.locator('[id^="add-to-cart"]')};
+
+    get sortDropdown() {return this.page.locator('.product_sort_container')};      
+   
+
+   async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
-    }
-}
+     } 
+   async addRandomProductsToCart() {
+          const inventories = await this.addItemToCartButton.all();  
+          const randomNum =  Math.max(1, Math.floor(Math.random() * inventories.length));; 
+          const addedInventories = [];  
+          const randomIndex = Array.from({ length: randomNum }, () => 
+               Math.floor(Math.random() *inventories.length)
```

Comment:
I would rather pick 2 random non-equal numbers, otherwise you should add logic to check is you picked one index twice. It could be something like 
```suggestion
               const randomIndexesArray = [];
               const expectedIndexesCount = 2;
               // checking if we have enough inventories to fill an array with unique indexes:
               while (randomIndexesArray.length < expectedIndexesCount && inventories.length >= expectedIndexesCount ) {
                   const randomIndex = Math.floor(Math.random() *inventories.length);
                   if (randomIndexesArray.indexOf(randomIndex) < 0) randomIndexesArray.push(randomIndex)
               }
```